media-video/gaupol/gaupol-0.91.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{3_3,3_4,3_5} )
+
+inherit distutils-r1 gnome2-utils virtualx xdg
+
+DESCRIPTION="A subtitle editor for text-based subtitles"
+HOMEPAGE="http://otsaloma.io/gaupol/"
+SRC_URI="https://github.com/otsaloma/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="spell test"
+
+RDEPEND="app-text/iso-codes
+	dev-python/chardet[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	x11-libs/gtk+:3[introspection]
+	spell? (
+		>=dev-python/pyenchant-1.4[${PYTHON_USEDEP}]
+		app-text/gtkspell:3
+	)"
+DEPEND="${RDEPEND}
+	dev-util/intltool
+	sys-devel/gettext
+	test? (
+		dev-python/pytest[${PYTHON_USEDEP}]
+		dev-python/pytest-runner[${PYTHON_USEDEP}]
+	)
+"
+
+DOCS=( AUTHORS.md NEWS.md TODO.md README.md README.aeidon.md )
+
+src_prepare() {
+	xdg_src_prepare
```

Comment:
To expand on that: `distutiils-r1_src_configure` already calls `xdg_environment_reset`, making xdg_src_prepare entirely redundant.